foundations/html_css/html-foundations/elements-and-tags.md

```diff
@@ -50,3 +50,4 @@ This section contains helpful links to related content. It isn’t required, so
 
 *   [Don't Fear the Internet's video about HTML](http://www.dontfeartheinternet.com/02-html)
 
+<br/>
```

Comment:
For all of the files that have been updated, these `<br />` tags still need to be removed.